source/_docs/tools/quick-open.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Quick Open Dialog"
+description: "Dialog for quickly accessing entities or running common commands"
+---
+
+You can access the "Quick Open Command Bar" dialog (similar to VSCode or Chrome Devtools) from anywhere in Home Assistant using hotkeys.
+
+<p class='img'>
+  <img src='/images/docs/quick-open/quick-open-dialog.gif' alt='Quick open dialog'>
+  Quick open bar for accessing entities and running commands
+</p>
+
+## Hotkeys
+
+| Mode  | Hotkey (Mac) | Hotkey (Win)
```

Comment:
```suggestion
| Mode  | Hotkey (Windows) | Hotkey (macOS)
```
These table headings were swapped causing it to show the wrong hotkeys underneath. And I think it's a little more professional to not abbreviate the OS.